aidbox/values.yaml

```diff
@@ -10,6 +10,9 @@ config:
   BOX_METRICS_PORT: 8765
   PGPORT: 5432
 
+auth:
+  existingSecret: ""
```

Comment:
there is `.Values.extraEnvFromSecrets` and it should cover external like secrets

similar approach could be applied to the AidboxDB

Reply:
Sure, I can update the PR to only add `extraEnvFromSecrets` to the db chart if that’s what you prefer. My concern is that we’re still creating an empty secret, which could add confusion when verifying which secrets are in use. That’s why I leveraged the `auth.existingSecret` logic commonly found in mainstream Helm charts (like Bitnami). But I’ll go with your call!